src/content/docs/accounts/original-accounts-billing/original-users-roles/parent-child-account-structure.mdx

```diff
@@ -94,7 +94,7 @@ How to set up SAML SSO will depend on which [user model](/docs/accounts/original
 **Requirements:**
 
 * Pro or Enterprise [edition](https://newrelic.com/pricing).
-* Your must be on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (if you're on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), you need assistance from New Relic).
+* Your must be on our [original user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models) (if you're on our [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), you need our assistance).
```

Comment:
"Your" --> "You"

Reply:
It seems weird to say "you need our assistance" without telling someone how to get that assistance.